lib/Controller/DraftsController.php

```diff
@@ -121,9 +121,9 @@ public function create(
 	 * @param string $editorBody
 	 * @param bool $isHtml
 	 * @param bool $failed
-	 * @param array $to i. e. [['label' => 'Linus', 'email' => 'tent@stardewvalley.com'], ['label' => 'Pierre', 'email' => 'generalstore@stardewvalley.com']]
-	 * @param array $cc
-	 * @param array $bcc
+	 * @param array<int, string[]>|array $to i. e. [['label' => 'Linus', 'email' => 'tent@stardewvalley.com'], ['label' => 'Pierre', 'email' => 'generalstore@stardewvalley.com']]
+	 * @param array<int, string[]>|array $cc
+	 * @param array<int, string[]>|array $bcc
```

Comment:
```suggestion
	 * @param array<int, string[]> $bcc
```

or the array can contain any types